tests/test_examples.py

```diff
@@ -108,38 +109,6 @@ def test_relative_binding():
             assert isinstance(result.dG_bind_err, float)
 
 
-def test_smc_biphenyl():
```

Comment:
Should we replace this test with a test for the `run_smc_on_freesolv.py`?

Reply:
This is tested below (test_smc_freesolv)